tests/moduletool/test_install.py

```diff
@@ -16,44 +16,161 @@
     Contact: code@inmanta.com
 """
 import argparse
+import configparser
 import json
 import os
+import re
 import shutil
 import subprocess
+from dataclasses import dataclass
 from importlib.abc import Loader
 from itertools import chain
-from typing import List, Optional, Tuple
+from typing import Dict, Iterator, List, Optional, Sequence, Set, Tuple, Union
 
 import py
 import pytest
 import yaml
+from pkg_resources import Requirement
 
-from inmanta import env, loader, module
+from inmanta import const, env, loader, module
 from inmanta.ast import CompilerException
 from inmanta.config import Config
 from inmanta.module import ModuleLoadingException
-from inmanta.moduletool import ModuleTool, ProjectTool
+from inmanta.moduletool import DummyProject, ModuleConverter, ModuleTool, ProjectTool
+from libpip2pi.commands import dir2pi
 from moduletool.common import BadModProvider, install_project
+from packaging import version
 
 
-def run_module_install(python_path: str, module_path: str, editable: bool, set_path_argument: bool) -> None:
+@pytest.fixture
+def build_venv_active(tmpvenv_active: Tuple[py.path.local, py.path.local]) -> Iterator[Tuple[py.path.local, py.path.local]]:
     """
-    Install the Inmanta module (v2) using the `inmanta module install` command.
+    Yields an active virtual environment that is suitable to build modules with.
+    """
+    env.process_env.install_from_index([Requirement.parse("virtualenv")])
```

Comment:
do we still use `virtualenv`? I assumed we where on `venv` completely? 

(I am quite certain we should be, or all hell would break loose)

Reply:
It's the `build` package that requires it. Installing `build` instead seems to do the trick as well, that's cleaner.

Reply:
scared me there